Header header
float32[] X
float32[] Y
float32[] Z
bool appendRefresh
int32 nPoints
float32 samplingTime
